pkg/combustion/kubernetes_test.go

```diff
@@ -556,9 +1114,11 @@ func TestConfigureKubernetes_SuccessfulMultiNodeRKE2Cluster(t *testing.T) {
 	assert.Contains(t, contents, "cp $CONFIGFILE /etc/rancher/rke2/config.yaml")
 	assert.Contains(t, contents, "if [ \"$HOSTNAME\" = node1.suse.com ]; then")
 	assert.Contains(t, contents, "echo \"192.168.122.100 api.cluster01.hosted.on.edge.suse.com\" >> /etc/hosts")
+	assert.Contains(t, contents, "echo \"fd12:3456:789a::21 api.cluster01.hosted.on.edge.suse.com\" >> /etc/hosts")
 	assert.Contains(t, contents, "export INSTALL_RKE2_ARTIFACT_PATH=$ARTEFACTS_DIR/kubernetes/install")
 	assert.Contains(t, contents, "sh $ARTEFACTS_DIR/kubernetes/install-kubernetes.sh")
 	assert.Contains(t, contents, "systemctl enable rke2-$NODETYPE.service")
+	assert.NotContains(t, contents, "sh set-node-ip.sh")
```

Comment:
I'd suggest to go through all cases and ensure assertion for `set-node-ip` exists.